ompi/mca/osc/sm/osc_sm.h

```diff
@@ -18,6 +18,20 @@
 
 #include "opal/mca/shmem/base/base.h"
 
+#if OPAL_HAVE_ATOMIC_MATH_64
+
+typedef uint64_t osc_sm_post_type_t;
+#define OSC_SM_POST_BITS 6
+#define OSC_SM_POST_MASK 0x3f
+
+#else
+
+typedef uint32_t osc_sm_post_type_t;
+#define OSC_SM_POST_MASK 0x1f
+#define OSC_SM_POST_BITS 5
```

Comment:
the pedantic part of me is irritate that the order of the constants is different between the two cases...

Reply:
fixed.